external/CMakeLists.txt

```
@@ -325,6 +325,10 @@ target_include_directories(libatrac9 PUBLIC LibAtrac9/C/src)
 
 option(BUILD_SHARED_LIBS "Build shared library" OFF)
 option(XXHASH_BUILD_XXHSUM "Build the xxhsum binary" OFF)
+option(XXH_X86DISPATCH_ALLOW_AVX "Allow building XXH3 with AVX even if it crashes on SSE2-Only CPUs" OFF)
+if(XXH_X86DISPATCH_ALLOW_AVX)
+	add_definitions(-DXXH_X86DISPATCH_ALLOW_AVX)
```

```suggestion
	target_compile_definition(xxhash PRIVATE XXH_X86DISPATCH_ALLOW_AVX)
```
We don't want to add this definition to every file in the project if it is only used in a single cpp file (xxh_x86dispatch.c)